_docs/overview/cloudgov-benefits.md

```diff
@@ -28,6 +28,8 @@ cloud.gov enforces an immutable infrastructure. Instead of logging into a live s
 
 cloud.gov has a [Provisional Authority to Operate (P-ATO) at the Moderate impact level from the FedRAMP Joint Authorization Board (JAB)]({{ site.baseurl }}{% link _docs/overview/fedramp-tracker.md %}). When you deploy a system on cloud.gov, you can [leverage this P-ATO](https://www.fedramp.gov/faqs/) as part of your agency ATO.
 
+Of the [261 security controls](https://nvd.nist.gov/800-53/Rev4/impact/moderate) required for FISMA Moderate-impact systems, as many as 60 percent of the controls covered as part of the cloud.gov P-ATO can be partially or fully inherited for agency systems deployed on the cloud.gov platform. The decision of which controls apply to an IT system and/or which may be inherited from the underlying platform ultimately rests with your agency's Authorizing Official.  However, this inheritability of controls can dramatically shorten the time and effort required to obtain an ATO for your system.
```

Comment:
Needs plainer language with shorter sentences and active voice,  e.g.:

FISMA moderate-impact systems require compliance with 261 controls. You can fully- or partially-inherit 60 percent of those from the cloud.gov P-ATO.

Reply:
I took a whack at revising this paragraph.